### PR TITLE
Update how-to-update-handbook.md

### PR DESCRIPTION
**Pull request by natalie-hub**
Added instruction to message DevOps team channel for GitHub access.

### CODE REVIEW

**Review by jaydeland — COMMENTED** [review]
Is this for contributors instead of FTE?

File: company/how-to-guides-for-staff/how-to-update-handbook.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 ## Steps
 
-If this is your first time contributing to Mattermost, first read the [Mattermost Contributor Agreement](https://www.mattermost.org/mattermost-contributor-agreement/) and sign it \(at the bottom of the page\), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true). Please ensure the **GitHub username** field matches your GitHub username exactly, including capitalization.
+If this is your first time contributing to Mattermost, first read the [Mattermost Contributor Agreement](https://www.mattermost.org/mattermost-contributor-agreement/) and sign it \(at the bottom of the page\), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true). Please ensure the **GitHub username** field matches your GitHub username exactly, including capitalization. Next, please request Mattermost GitHub access by using the [DevOps Team Channel](https://community.mattermost.com/private-core/channels/devops-team).
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-update-handbook.md]:
```suggestion
If this is your first time contributing to Mattermost, first read the [Mattermost Contributor Agreement](https://www.mattermost.org/mattermost-contributor-agreement/) and sign it \(at the bottom of the page\), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true). Please ensure the **GitHub username** field matches your GitHub username exactly, including capitalization. Next, please request Mattermost GitHub access in the [DevOps Team Channel](https://community.mattermost.com/private-core/channels/devops-team).
```

**Review by justinegeffen — DISMISSED** [review]

**Review by natalie-hub — COMMENTED** [review]
TY!

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
@natalie-hub, is this for contributors or FT staff?

**Comment by justinegeffen** [comment]
@natalie-hub, gentle reminder on this. :)

**Comment by justinegeffen** [comment]
Following up on this, @natalie-hub. :)

**Review by justinegeffen — DISMISSED** [review]

**Comment by justinegeffen** [comment]
Re-reading the content and location, it's under staff guides so the instructions are likely for FT staff, @jaydeland.

File: company/how-to-guides-for-staff/how-to-update-handbook.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 ## Steps
 
-If this is your first time contributing to Mattermost, first read the [Mattermost Contributor Agreement](https://www.mattermost.org/mattermost-contributor-agreement/) and sign it \(at the bottom of the page\), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true). Please ensure the **GitHub username** field matches your GitHub username exactly, including capitalization.
+If this is your first time contributing to Mattermost, first read the [Mattermost Contributor Agreement](https://www.mattermost.org/mattermost-contributor-agreement/) and sign it \(at the bottom of the page\), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true). Please ensure the **GitHub username** field matches your GitHub username exactly, including capitalization. Next, please request Mattermost GitHub access in the [DevOps Team Channel](https://community.mattermost.com/private-core/channels/devops-team).
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-update-handbook.md]:
```suggestion
If this is your first time contributing to Mattermost, first read the [Mattermost Contributor Agreement](https://www.mattermost.org/mattermost-contributor-agreement/) and sign it \(at the bottom of the page\), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true). Please ensure the **GitHub username** field matches your GitHub username exactly, including capitalization. Next, please request access to the Mattermost organization [DevOps Team Channel](https://community.mattermost.com/private-core/channels/devops-team).
```

**Review by jaydeland — DISMISSED** [review]

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**